src/main/java/prefs/UserConfig.java

```diff
@@ -0,0 +1,7 @@
+package prefs;
+
+/**
+ * Abstraction for user-defined settings.
+ */
+public class UserConfig {
+}
```

Comment:
Remind me. Why do we have this? I though we decided not to have it until we actually need it.